src/templates/extends.ejs

```diff
@@ -5,5 +5,5 @@
 <%= options.extend.map(({ fullname }) =>`import ${'extend_' + hash(fullname)} from '${fullname.replace(/\\/g,'/')}';`).join('\n') %>
 
 export default {
-<%= options.extend.map(({ fullname }) =>`\t\textend_${hash(fullname)},`).join('\n') %>
+<%= options.extend.map(({ fullname }) =>`\textend_${hash(fullname)},`).join('\n') %>
```

Comment:
With double `\t` the output build file has wrong formatting:
Instead of this:
```
let test = [
   'a'
]
```

with double `\t`:
```
let test = [
      'a'
]
```
Two tabs are created.